README.md

```diff
@@ -41,21 +40,20 @@ target_include_directories(
     PRIVATE "${CMAKE_CURRENT_BINARY_DIR}"
 )
 
-# To use tree-gen's support library (you could substitute your own if you feel
-# up to the task):
+# To use tree-gen's support library (you could substitute your own if you feel up to the task):
 target_link_libraries(my-software tree-lib)
 ```
 
 and CMake *Should*™ handle everything for you.
 
 `tree-gen` does have some dependencies:
 
- - A compiler with C++11 support (MSVC, GCC, and Clang are tested in CI);
+ - A compiler with C++11 support (MSVC, GCC, and Clang are tested in CI)
```

Comment:
I think this becomes C++2x no?
Can fmt work with C++11?

Reply:
Yes, `fmt` is a C++11 library. Actually, it doen't rely on many C++11 features (see [here](https://fmt.dev/latest/index.html#portability)).

You can also see that tree-gen's `CMakeLists.txt` still uses C++11 standard.